lib/ramble/docs/tutorials/using_zips_and_matrices.rst

```diff
@@ -0,0 +1,354 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _zips_and_matrices_tutorial:
+
+==============================
+7) Zips and Matrices
```

Comment:
Is this techincally Zips, Vectors, and Matrices? or are you OK with the zip implies vector?

Reply:
Vectors are already used in the basic scaling study tutorial. So, this is more specifically `Explicit Zips and Matrices`.